smithy-aws-protocol-tests/model/restJson1/http-headers.smithy

```diff
@@ -46,6 +46,22 @@ apply InputAndOutputWithHeaders @httpRequestTests([
             headerStringSet: ["a", "b", "c"],
         }
     },
+    {
+        id: "RestJsonInputAndOutputWithQuotedStringHeaders",
+        documentation: "Tests requests with string list header bindings that require quoting",
+        protocol: restJson1,
+        method: "POST",
+        uri: "/InputAndOutputWithHeaders",
+        headers: {
+            "X-StringList": """
+            a, "b,c", "\"def\""
+            """
```

Comment:
This should go on the previous line. By putting it on the next line, it's inserting a "\n".

Reply:
Yes, but if I turn that into:

```
        headers: {
            "X-StringList": """
            a, "b,c", "\"def\"""""
        },
```

Then the entire file doesn't parse. I can work around this by flipping the order of the parameters, or by adding a whitespace (which is ignored according to spec)